### ISSUE TITLE
Добавить апи для получения событий

### ISSUE THREAD

**Issue by Shiplayer** [codex-team/hawk.android#53]
- [ ] События должны получаться по указанному проекту

**Comment by temnik15** [comment]
Здесь продолжаем цепь WorkspaceP->ProjectP->EventP?

**Comment by Shiplayer** [comment]
нет, там было необходимо, так как мы никак не можем получить отдельный список проектов по указаному воркспейсу

**Comment by temnik15** [comment]
А, не заметил, там вы не запрашиваете всю информацию о проекте. ок
